internal/testing/cluster_test.go

```diff
@@ -269,6 +271,139 @@ func TestNewTestCluster(t *testing.T) {
 	}
 }
 
+// TestObjectEntriesIntegration is an integration test that verifies objects are
+// uploaded, download and deleted from and to the paths we would expect. It is
+// similar to the TestObjectEntries unit test, but uses the worker and bus
+// client to verify paths are passed correctly.
+func TestObjectEntriesIntegration(t *testing.T) {
```

Comment:
I think we don't need `Integration` here since all tests in `testing` are integration tests.

Reply:
Ok, wanted to avoid having `TestObjectEntries` twice